### PR TITLE
Use custom labels always, fix SGE loading

### PR DESCRIPTION
**Pull request by bdvorachek**
# Critical Changes

# Changes
- Always display the label defined in the template when rendering fields
- Fixed an issue where the form did not load in single gift entry mode
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

**lurch: attach W-038207

### CODE REVIEW

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HpwXQAS" target="_blank">W-038207</a>

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
 
                 // get the target field names to be used by getRecord
                 this.fieldNames = getRecordFieldNames(this.formTemplate, this.fieldMappings);
+                this.initializeForm(this.formTemplate);
```
Comment (vmartinov-sforg) [inline src/lwc/geFormRenderer/geFormRenderer.js]: @bdvorachek Did you check if the contact / account field values still display after adding this line?

Comment (bdvorachek) [reply src/lwc/geFormRenderer/geFormRenderer.js]: I've updated this to handle the case where we do have a donor record id present and need to load existing data. If there is one present, the initializeForm call should happen during the wiredGetRecord for the donor.

Comment (vmartinov-sforg) [reply src/lwc/geFormRenderer/geFormRenderer.js]: I'm getting an exception, it looks like isEmpty() is being used in utilTemplateBuilder, but it was moved to utilCommon. It needs to be imported.

Uncaught (in promise) ReferenceError: LWC component's @wire target property or method threw an error during value provisioning. Original error:
[isEmpty is not defined]
    at Object.e.setRecordValuesOnTemplate (/lightning/n/modules/c/utilTemplateBuilder.js:4)

**Pull request merged**